solr/modules/ltr/src/test/org/apache/solr/ltr/store/rest/TestModelManagerPersistence.java

```diff
@@ -137,6 +139,24 @@ public void testFeaturePersistence() throws Exception {
     assertJQ(ManagedModelStore.REST_END_POINT, "/models/==[]");
   }
 
+  @Test
+  public void testModelIsStoredCompact() throws Exception {
```

Comment:
minor: rename to reflect that not just model storage compactness is tested

```suggestion
  public void testIsStoredCompact() throws Exception {
```

or

```suggestion
  public void testFeaturesAndModelAreStoredCompact() throws Exception {
```